### PR TITLE
Added tests for Android Wear app [US #95] [Task #99]

### PR DESCRIPTION
**Pull request by miisaka**
### WHAT kind of change does this PR introduce?
Added UI tests for the android wear app

### HOW is this accomplished??
By testing all the arithmetic operations on the Android Wear app

### Checklist
- [x] Docs have been added / updated to reflect the changes
- [x] I have reviewed and tested the changes :white_check_mark:
- [x] I have squashed my commits to have a reasonable amount of commits
- [x] All of the code I have written adhere to the coding conventions outlined in the coding conventions section of the wiki

### CODE REVIEW

File: wear/src/androidTest/java/com/simpletools/calculator/wear/tests/MainActivityTest.java [hunk 1]
```diff
@@ -45,6 +47,129 @@ public void removeLeadingZero() {
...
+        press(R.id.btn_1);
+        press(R.id.btn_0);
+        pressOperation("÷");
+        press(R.id.btn_4);
+        press(R.id.btn_equals);
+        checkResult("2.5");
+    }
+
+    @Test
+    public void divisionByZeroTest() {
```
Comment (almiche) [inline wear/src/androidTest/java/com/simpletools/calculator/wear/tests/MainActivityTest.java]: Nice testing of edge cases !

**Comment by almiche** [comment]
✅ Styling is adequate and follows coding conventions outlined by the Kotlin Foundation

✅ New methods and functionalities have appropriate tests (testing edge cases + good coverage)

✅ Branch has an acceptable amount of commits with descriptive names and

✅ PR description matches the code outlined a has mentions to issues

✅ This PR attempts to implement a specific feature (no code blob or epic pr)

Status: **Approved**

Comments: Tests are thorough looking good !
